mlflow/pyfunc/__init__.py

```diff
@@ -709,7 +715,7 @@ def save_model(
     artifacts=None,
     signature: ModelSignature = None,
     input_example: ModelInputExample = None,
-    **kwargs
+    **kwargs,
```

Comment:
@wentinghu Can you revert this change because this causes a syntax error in Python 3.5?